cromwell_tools/cli.py

```diff
@@ -68,10 +69,10 @@ def add_auth_args(subcommand_parser):
                         help='Whether to submit the workflow in "On Hold" status.')
 
     # wait arguments
-    wait.add_argument('workflow-ids', nargs='+')
-    wait.add_argument('--timeout-minutes', type=int, default=120,
+    wait.add_argument('workflow_ids', nargs='+')
```

Comment:
How about keeping this argument `'workflow-ids'` and then setting `dest='workflow_ids'` for consistency with the other arguments?

Reply:
Good point! So this is the feature of argparse that I don't like: if you add `dest='workflow_ids'` to this positional argument, it will raise an error like:
```
Traceback (most recent call last):
  File "/Users/xxx/SDK/Py3Sci/bin/cromwell-tools", line 11, in <module>
    sys.exit(main())
  File "/Users/xxx/SDK/Py3Sci/lib/python3.6/site-packages/cromwell_tools/cli.py", line 111, in main
    command, args = parser(arguments)
  File "/Users/xxx/SDK/Py3Sci/lib/python3.6/site-packages/cromwell_tools/cli.py", line 72, in parser
    wait.add_argument('workflow-ids', dest='workflow_ids', nargs='+')
  File "/usr/local/Cellar/python/3.6.4_4/Frameworks/Python.framework/Versions/3.6/lib/python3.6/argparse.py", line 1315, in add_argument
    raise ValueError('dest supplied twice for positional argument')
ValueError: dest supplied twice for positional argument
```
It's because `argparse` already added `dest='workflow_ids'` for you under the hood, which means the argument name here must be strictly consistent with the API's function argument, we don't have much ability to control this behavior :(